a3m/client/clientScripts/assign_file_uuids.py

```diff
@@ -79,8 +79,6 @@ def assign_transfer_file_uuid(
     )
     event_type = "ingestion"
     file_uuid = str(uuid.uuid4())
-    job.print_output(f"Generated UUID for file {file_uuid}")
```

Comment:
Not sure if I understand your decision here. If we stop printing messages then they won't be captured at all, but I think there's still value in that, right? It may be capture as part of the output task and put in the datatabase for later revision or reporting if I remember correctly, and that seems to be useful.